src/style.css

```diff
@@ -0,0 +1,166 @@
+:root {
+  --star: url(images/star.svg);
+  --star-active: url(images/star-active.svg);
+  --width: 200px;
+}
+
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  width: var(--width);
+  height: 403px;
+  padding-top: 30px;
+  padding-bottom: 15px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  border-radius: 5px;
+}
+
+.card > :nth-child(1) {
+  height: 134px;
+  width: calc(var(--width) * 0.8);
+  display: block;
+  margin: 0 auto 45px;
+}
+
+.card__description {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+  width: calc(var(--width) * 0.85);
+  margin: 0 auto 5px;
+}
+
+.card__code {
+  font-size: 11px;
+  width: calc(var(--width) * 0.85);
+  margin: 0 auto 16px;
+}
+
+.card__price,
+.card__rating {
+  display: flex;
+  justify-content: space-between;
+  width: calc(var(--width) * 0.85);
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.card__rating {
+  align-items: flex-start;
+  margin-bottom: 20px;
+}
+
+.stars-block {
+  display: flex;
+  margin-top: 1px;
+}
+
+.stars-block > * {
+  height: 13px;
+  width: 13px;
+  background-repeat: no-repeat;
+  margin: 0 3px;
+}
+
+.stars-block > :first-child {
+  margin-left: 1px;
+}
+
+.stars--0 > * {
+  background-image: var(--star);
+}
+
+.stars--1 :nth-child(n) {
+  background-image: var(--star-active);
+}
+
+.stars--1 :nth-child(n + 2) {
+  background-image: var(--star);
+}
+
+.stars--2 :nth-child(n) {
+  background-image: var(--star-active);
+}
+
+.stars--2 :nth-child(n + 3) {
+  background-image: var(--star);
+}
+
+.stars--3 :nth-child(n) {
+  background-image: var(--star-active);
+}
+
+.stars--3 :nth-child(n + 4) {
+  background-image: var(--star);
+}
+
+.stars--4 :nth-child(n) {
+  background-image: var(--star-active);
+}
+
+.stars--4 :nth-child(5) {
+  background-image: var(--star);
+}
+
+.stars--5 > * {
+  background-image: var(--star-active);
+}
+
+.card__feedback {
+  font-size: 11px;
+  text-align: right;
+  font-weight: 400;
+  color: #000;
+}
+
+.card__price {
+  height: 15px;
+}
+
+.card__price > :nth-child(1) {
+  margin-top: 5px;
+  font-size: 11px;
+  line-height: 13px;
+  text-align: right;
+}
+
+.card__price > :nth-child(2) {
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 19px;
+  text-align: right;
+}
+
+.card__button {
+  display: flex;
+  margin: 0 auto;
+  justify-content: center;
+  align-items: center;
+  height: calc(var(--width) * 0.2);
+  width: calc(var(--width) * 0.85);
+  background: #00acdc;
+  border-radius: 5px;
+  text-decoration: none;
+  font-weight: bold;
+  font-size: 14px;
+  color: #fff;
+  border: 1px solid #00acdc;
+  text-transform: uppercase;
+  margin-top: 25px;
+}
+
+.card__button:hover {
+  color: #00acdc;
+  background: #fff;
+  padding-bottom: 1px;
+  margin-top: 24px;
```

Comment:
don't add anything which can change the size of the button, when the user hovers on it, I mean real size, transform: scale, for example, it's okay